entity-framework/core/what-is-new/ef-core-6.0/breaking-changes.md

```diff
@@ -133,7 +133,7 @@ Instead of assigning the same owned instance to a new owner you can assign a cop
 ### Cosmos: Related entity types are discovered as owned
 
 [Tracking Issue #24803](https://github.com/dotnet/efcore/issues/24803)
-[What's new: Default to implicit ownership](/core/what-is-new/ef-core-6.0/whatsnew#default-to-implicit-ownership)
+[What's new: Default to implicit ownership](whatsnew.md#default-to-implicit-ownership)
```

Comment:
Should use xref: syntax in all 3.

Reply:
I changed links to xref syntax. Altough I have to say that I don't understand the reason why use xref syntax when the other syntax is usually chosen in markdown. And article [How to use links in documentation](https://docs.microsoft.com/en-us/contribute/how-to-write-links#xref-cross-reference-links) says that xref links are recommended when linking to APIs. And xref links don't work when browsing markdowns directly on github.

Reply:
They are validated at runtime and allow to reference other docsets. Also they are protected against file renames. We have decided to use xref syntax rather than ".md" files specifically for the latter. A lot of things in the doc page cannot be rendered in markdown mode (API docs/ code samples/ tabs), so don't see a reason to try hard to preserve these links.